### DISCUSSION TITLE
Println doesn't showing in terminal

### DISCUSSION THREAD

**Discussion by tonnax** [pocketbase/pocketbase#3847] [Q&A] [answered]
![image](https://github.com/pocketbase/pocketbase/assets/140943390/978f8063-f2d1-4e87-84d5-0f68c52a7e52)
Hi! Sorry for the newbie question, after upgrading to Pocketbase 19.4 from 17.7 I can NO more using the command `go run main.go serve` to run my Pocketbase Go framework and server since it will return error as shown in the above image.

But using `./pocketbase serve --debug` works, I can run the server on my local. The problem is that now my `fmt.println` and `log.println` are **NOT SHOWING** in the terminal where my server is serve, it only shows db operations details. Previously with `go run main.go serve` it shows well. I need the println to help me debug the code. Thanks in advance.

Appendix: 
what I mean by 'db operations details': 
![image](https://github.com/pocketbase/pocketbase/assets/140943390/76bedfb4-9f35-4f11-80e4-c038d3619b91)

**Comment by ganigeorgiev** [comment, marked as answer]
The above error doesn't seem to be PocketBase related but from a recent breaking change in `aws-sdk-go-v2` package (see https://github.com/google/go-cloud/issues/3341#issuecomment-1821300321).

To fix it try with downgrading your github.com/aws/aws-sdk-go-v2 go dependency to the one from the [go.mod](https://github.com/pocketbase/pocketbase/blob/master/go.mod#L35) that the prebuilt executable is using. For example:

```
go get github.com/aws/aws-sdk-go-v2@v1.21.2
```

(or change it manually and run `go mod tidy`)

**Reply by tonnax** [reply]
you rock @ganigeorgiev !!!!!
I will pay you back one day for this awesome invention of yours! btw I will keep my best not asking too much question here before searching and reading all the Q&As. Thanks a bunch! 🥳
